projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -277,6 +362,28 @@ public function checkout_process() {
 	public function order_process( $order_id ) {
 		$order = wc_get_order( $order_id );
 
+		$payment_option = $order->get_payment_method();
+
+		if ( is_object( WC()->session ) ) {
+			$create_account = true === WC()->session->get( 'wc_checkout_createaccount_used' ) ? 'Y' : 'N';
+		} else {
+			$create_account = 'N';
+		}
+
+		$guest_checkout = $order->get_user() ? 'N' : 'Y';
```

Comment:
Let's standardise the format of boolean props (`1 0`, `Y N`, `yes no` etc). 

Note that elsewhere in this file, `1 0` is used for boolean props (e.g. `cart_page_contains_cart_block`). This is compact and unambiguous; I'd recommend sticking with this, or whatever data team recommends.